{"links": {"self": {"href": "data/repositories/osrf/sandia-hand/issues/9/comments/5151212.json"}, "html": {"href": "#!/osrf/sandia-hand/issues/9#comment-5151212"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/sandia-hand/issues/9.json"}}, "type": "issue", "id": 9, "repository": {"links": {"self": {"href": "data/repositories/osrf/sandia-hand.json"}, "html": {"href": "#!/osrf/sandia-hand"}, "avatar": {"href": "data/bytebucket.org/ravatar/{ffa260f4-e54c-4b4b-ab64-4d0fc5a90091}ts=default"}}, "type": "repository", "name": "sandia-hand", "full_name": "osrf/sandia-hand", "uuid": "{ffa260f4-e54c-4b4b-ab64-4d0fc5a90091}"}, "title": "simple_grasp.py in sandia_hand_teleop isn't installed"}, "content": {"raw": "Ah, thanks for pointing that out. I had recently tried to clean up sandia_hand_teleop a bit and in the process I broke that. 61622ce should fix it.", "markup": "markdown", "html": "<p>Ah, thanks for pointing that out. I had recently tried to clean up sandia_hand_teleop a bit and in the process I broke that. <a href=\"#!/osrf/sandia-hand/commits/61622ce\" rel=\"nofollow\" class=\"ap-connect-link\">61622ce</a> should fix it.</p>", "type": "rendered"}, "created_on": "2013-07-11T03:06:56.415944+00:00", "user": {"display_name": "Morgan Quigley", "uuid": "{92f5a4e9-2e82-42b7-955b-3eda9a272364}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B92f5a4e9-2e82-42b7-955b-3eda9a272364%7D"}, "html": {"href": "https://bitbucket.org/%7B92f5a4e9-2e82-42b7-955b-3eda9a272364%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/c40268cda48633a945ef88a3f37da8dcd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMQ-4.png"}}, "nickname": "Morgan Quigley", "type": "user", "account_id": "557058:6d82cad4-6685-41dd-9e43-f2af8c287dbb"}, "updated_on": null, "type": "issue_comment", "id": 5151212}